packages/core/src/components/sections/Hero/Hero.tsx

```diff
@@ -36,16 +33,22 @@ const Hero = ({
   title,
   subtitle,
   image,
-  variant = HeroWrapper.props.variant ?? 'primary',
-  colorVariant = HeroWrapper.props.colorVariant ?? 'main',
+  variant = 'primary',
+  colorVariant = 'main',
   icon,
 }: HeroProps) => {
+  const {
+    Hero: HeroWrapper,
+    HeroImage,
+    HeroHeader,
+  } = useOverrideComponents<'Hero'>()
+
   return (
     <Section className={`${styles.section} section-hero`}>
       <HeroWrapper.Component
         {...HeroWrapper.props}
-        variant={variant}
-        colorVariant={colorVariant}
+        variant={HeroWrapper.props.variant ?? variant}
```

Comment:
This order has to be changed since we always prioritize the value that comes from the CMS (prop).
In this example, line 36 cannot contain the default definition for the variant prop, or else it'd be impossible to tell apart the CMS value from our default value.

```suggestion
        variant={variant ?? HeroWrapper.props.variant ?? 'primary'}
```

Reply:
Thanks for that. I was wondering what would happen and that's why I inverted the order here. I'll fix the function arguments 🙏